Content/LeagueSandbox-Scripts/Champions/Yasuo/Q1.cs

```diff
@@ -0,0 +1,79 @@
+using System.Numerics;
+using System.Linq;
+using GameServerCore.Enums;
+using GameServerCore.Domain.GameObjects;
+using static LeagueSandbox.GameServer.API.ApiFunctionManager;
+using GameServerCore.Domain;
+using LeagueSandbox.GameServer.Scripting.CSharp;
+using GameServerCore;
+
+namespace Spells
+{
+    public class YasuoQW : IGameScript
+    {
+        public void OnActivate(IChampion owner)
+        {
+        }
+
+        public void OnDeactivate(IChampion owner)
+        {
+        }
+
+        public void OnStartCasting(IChampion owner, ISpell spell, IAttackableUnit target)
+        {
+            var current = new Vector2(owner.X, owner.Y);
+            var to = Vector2.Normalize(new Vector2(spell.X, spell.Y) - current);
+            var range = to * 475;

```

Comment:
Magic number: 475

Reply:
in the `YasuoQW.json` it's `"CastRangeDisplayOverride": "475.0000",`

Reply:
Replace as `spell.SpellData.CastRangeDisplayOverride[0]`